action.yml

```diff
@@ -62,6 +62,8 @@ runs:
         TF_ENV_VERSION: v3.0.0
         TG_SWITCH_VERSION: 0.6.0
       run: |
+        rm -rf ~/.tfenv
+        rm -rf "${HOME}/tgswitch"
```

Comment:
I saw this same issue and made a [PR](https://github.com/gruntwork-io/pipelines-execute/pull/6) as well. The issue only happened with tfenv, not tgswitch but i also can not explain why the folder exists but the executable does not.

Reply:
I will merge as-is for now. I need this change released.